CHANGELOG.md

```diff
@@ -13,7 +13,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Fixes
 
--
+-  Compliance Operator failed to un-puase MCP after remediation is applied, 
+   some KubeletConfig configuration are rendered into multiple files in
+   MachineConfig, and compliance operator failed to counter this situation.
```

Comment:
nit: Compliance Operator to be consistent with the usage above (and elsewhere in the change log).